CONTRIBUTING.md

```diff
@@ -38,9 +38,9 @@ With that in mind, the steps for adding a new API are:
 
 ### Test before you commit
 
-We have two automated tests running on [Travis](https://travis-ci.org/rust-lang/libc):
+We have two automated tests running on [Travis](https://travis-ci.com/rust-lang/libc):
```

Comment:
This needs updating to: https://dev.azure.com/rust-lang2/libc/_build?definitionId=1&_a=summary

Reply:
Should the Travis link be replaced with the Azure one? Also is libc-test using Azure as well? The linked repo still lists Travis in its readme.

Reply:
Yes, libc-test is using azure as well.